### PR TITLE
INHEART-155: Hot Fix issue - PR comments 

### PR DESCRIPTION
**Pull request by BeataKula**
- delete space on underline filter 
- hide filtering area after clicking

obraz

### CODE REVIEW

File: apps/mobile/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "react-native-safe-area-context": "*",
     "jest-each": "*",
     "react-native-vector-icons": "*",
-    "react-native-keyboard-aware-scroll-view": "*"
+    "react-native-keyboard-aware-scroll-view": "*",
+    "date-fns": "*"
```
Comment (pawelroguski) [inline apps/mobile/package.json:20]: date-nfs is not needed here

**Review by pawelroguski — APPROVED** [review]

File: apps/mobile/package.json (hunk 1 above)
Comment (pmusialik-intive) [reply apps/mobile/package.json:20]: isn't it included automatically?

Comment (pawelroguski) [reply apps/mobile/package.json:20]: maybe

Comment (pawelroguski) [reply apps/mobile/package.json:20]: @BeataKula leave it as it is

Comment (BeataKula) [reply apps/mobile/package.json:20]: automaticly probably...

**Pull request merged**